### PR TITLE
Boxplot script

### PR DESCRIPTION
**Pull request by CameroneN**
This PR adds the plotscript.py script,  which produces a boxplot (using matplotlib) from generated stats.csv files 

- Paths to stats files are input, and the resulting boxplot can be customised through matplotlib functionality

### CODE REVIEW

File: sim/plotscript.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+import numpy
+import glob
+import os
+from pathlib import Path
+import shutil
+
+
+dirs = glob.glob("/home/cameronnicholls/boxplots/OFAT/*/stats.csv")
+
+values = pd.DataFrame(index =[1,2,3,4,5,6,7,8,9,10])
```
Comment (afishman) [inline sim/plotscript.py:13]: `range(1, len(dirs)+1)` is more general

File: sim/plotscript.py [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    path = Path(directory)
+    sample_name = path.parts[-2]
+    branches.append(sample_name)
+    csv = pd.read_csv(directory)
+    values[sample_name] = csv['FN']
+
+print(branches)
+plt.boxplot(values, showfliers = True)
+# y = (0.8, 0.85, 0.9, 0.95, 1)
+#y = (0.9, 0.92, 0.94, 0.96, 0.98, 1)
```
Comment (afishman) [inline sim/plotscript.py:26]: delete the comments

File: sim/plotscript.py [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+plt.xticks(range(0, len(branches)),  branches, rotation=-20)
+#plt.yticks(y)
```
Comment (afishman) [inline sim/plotscript.py:28]: comment

File: sim/plotscript.py [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+    sample_name = path.parts[-2]
+    branches.append(sample_name)
+    csv = pd.read_csv(directory)
+    values[sample_name] = csv['FN']
+
+print(branches)
+plt.boxplot(values, showfliers = True)
+# y = (0.8, 0.85, 0.9, 0.95, 1)
+#y = (0.9, 0.92, 0.94, 0.96, 0.98, 1)
+plt.xticks(range(0, len(branches)),  branches, rotation=-20)
```
Comment (afishman) [inline sim/plotscript.py:27]: `plt.grid()`?

File: sim/plotscript.py [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+import matplotlib as mpl
+import matplotlib.pyplot as plt
+import numpy
+import glob
+import os
+from pathlib import Path
+import shutil
+
+
+dirs = glob.glob("/home/cameronnicholls/boxplots/OFAT/*/stats.csv")
```
Comment (afishman) [inline sim/plotscript.py:11]: I think this should be a function in `ofat.py` called `plot`

Ideally the function would take a list of preloaded dataframes as input. For starters though, can just be a path to the root of the ofat directory.

**Review by afishman — CHANGES_REQUESTED** [review]
Looks great! A few small things.

File: sim/plotscript.py (hunk 5 above)
Comment (CameroneN) [reply sim/plotscript.py:11]: Hey just to clarify, by this do you mean the whole script could be replaced with a function in `ofat.py` ?

Comment (afishman) [reply sim/plotscript.py:11]: yes, easier for everyone to use if it's  abstracted into a function. I think `ofat.py` is a sensible module to put it in for now.

**Pull request closed without merging**